[RHDHPAI-814] Update TechDocs (#77)
redhat-ai-dev/ai-lab-template: documentation/app/audio-to-text/index.md

<!-- Original Recipe README: https://github.com/containers/ai-lab-recipes/blob/main/recipes/audio/audio_to_text/README.md
-->

> # **Audio to Text Software Template**

Reviewer: just curious if marking it as bold like this in a header make any difference? `# ** text**` 🤔

Author: I found that in the TechDocs if they weren't bolded they kind of blended into the normal text. wdyt?

Reviewer: Its fine with me if it makes it appear bolder

## **Using The Template**

This AI Software Template provides you with some customization to alter the final generated result. Aside from being able to input your desired name for the application and container image, you can control:

- The owner listed through Red Hat Developer Hub (RHDH).
- The desired GitHub/GitLab organization and repository.
- The namespace the application and applicable model servers are deployed to.

!!! tip "Git Repositories"

    You can choose between GitHub and GitLab as your desired Source Code Management (SCM) platform, and the template fields will update accordingly!

This template provides the option to supply your own model and model server. By utilizing this option you will need to ensure that the models provided support Automatic Speech Recognition (ASR) tasks.

## **Deployable Application**

This AI Software Template will create a web application that utilizes the [ggerganov/whisper.cpp](https://huggingface.co/ggerganov/whisper.cpp) model to transform audio files into text. 

!!! info

>     This model is classified as an "Automatic Speed Recognition (ASR)" model. These models are specifically trained and designed to accurately transcribe spoken language into written text!

Reviewer: ```suggestion
    This Software Template uses an "Automatic Speed Recognition (ASR)" model. These models are specifically trained and designed to accurately transcribe spoken language into written text!
```
how does this sound?
